src/components/button-toggle/button-toggle-group/button-toggle-group.stories.tsx

```diff
@@ -167,3 +302,27 @@ export const AriaLabel: ComponentStory<typeof ButtonToggleGroup> = () => {
     </ButtonToggleGroup>
   );
 };
+
+export const AriaLabelMinor: ComponentStory<typeof ButtonToggleGroup> = () => {
```

Comment:
**isue:**: story is not used in `button-toggle.stories.mdx`

Reply:
**suggestion**: add story like so `groupStories.AriaLabelMinor` (even though i'm against this approach)